build.sbt

```diff
@@ -207,5 +246,44 @@ def project(name: String, plugged: AutoPlugin*) = {
     )
 }
 
-lazy val setReleaseVersions: String => State => State =
-  v => _.put(ReleaseKeys.versions, (v, v))
+lazy val bindingsExtraArgs = Try {
+  val version = "llvm-config --version".!!.trim
+  val libDir  = "llvm-config --libdir".!!.trim
+  s"-I$libDir/clang/$version/include"
+}.toOption
+
+def bindingProject(name: String) = {
```

Comment:
I'm not sure if we want to version all bindings with the same version as the rest of bindgen or allow to version according to the upstream version of the native library for example.

Reply:
Hm.. if we decide to set upstream version then we will not be to change bindings (for example code layout) until library version changes :(
Maybe we can use bindgen version and manually add a comment about upstream version?

Reply:
Yes, there are still a few things to flesh out. Right now the binding artifact names are prefixed with `lib`. We could publish them under another organization ID like `org.scalanative.bindgen.bindings` but I think that can be done separately once we have a better idea how we want to manage the bindings.